controllers/airline_controller.go

```diff
@@ -27,18 +30,35 @@ func (acr *AirlineControllerRepository) HandleGetAirline(ctx *gin.Context) {
 		ctx.JSON(400, gin.H{"msg": "Page number must be greater than 0"})
 		return
 	}
-	appAirline, err := acr.service.GetAirline(page)
+	airline, err := acr.service.GetAirline(page)
 	if err != nil {
 		ctx.JSON(http.StatusNotFound, gin.H{"error": "Airlines Details Not found"})
 	}
-	ctx.JSON(http.StatusOK, appAirline)
+	ctx.JSON(http.StatusOK, airline)
 }
 
 func (acr *AirlineControllerRepository) HandleGetAirlineById(ctx *gin.Context) {
 	airlineId := ctx.Param(`id`)
-	appAirline, err := acr.service.GetAirlineById(airlineId)
+	airline, err := acr.service.GetAirlineById(airlineId)
 	if err != nil {
 		ctx.JSON(http.StatusNotFound, "error: Incorrect Airlines Id")
 	}
-	ctx.JSON(http.StatusOK, appAirline)
+	ctx.JSON(http.StatusOK, airline)
+}
+
+func (acr *AirlineControllerRepository) HandleCreateNewAirline(ctx *gin.Context) {
+	var airline models.Airline
+
+	err := ctx.ShouldBindWith(&airline, binding.JSON)
```

Comment:
either in this PR or in another, please ensure consistency in using the method/pattern for all controllers instead of `BindJSON`

Reply:
yeah ...okay vijay..i ll inform all